backend/app/adapter/db/user_changelog.go

```diff
@@ -32,6 +32,10 @@ WHERE "%s"=$1;`,
 	row := u.db.QueryRow(statement, user.Email)
 	lastViewedAt := time.Time{}
 	err := row.Scan(&lastViewedAt)
+	if errors.Is(err, sql.ErrNoRows) {
+		return lastViewedAt, repository.ErrEntryNotFound("user does not exist")
```

Comment:
Should i change this to
` repository.ErrEntryNotFound("user not found")`

Reply:
Do it.